cs/ccxt/base/Exchange.cs

```diff
@@ -25,6 +25,9 @@ public Exchange(object userConfig2 = null)
             this.setMarkets(this.markets);
         }
         this.afterConstruct();
+        
+        var is_sandbox = this.safeBool2(this.options, "sandbox", "testnet", false);
+        this.setSandboxMode(is_sandbox);
```

Comment:
I would rather call it only when true, because setSandboxMode might be side-effects